build-support/bin/BUILD

```diff
@@ -11,7 +11,7 @@ files(
 # stripping the source root from the files.
 files(
   name = 'python_scripts',
-  sources = ['*.py'],
+  sources = ['common.py', 'check_header.py'],
```

Comment:
Without this change, most of the scripts have multiple owners, meaning file args don't work with them.